src/main/resources/grammar/Jakt.bnf

```diff
@@ -484,8 +484,10 @@ MatchCaseTrail ::= NL FAT_ARROW (Block | Expression) COMMA?
 MatchCaseHead ::= <<pipeList (MatchCaseElseHead | MatchPattern | Expression)>>
 MatchCaseElseHead ::= ELSE_KEYWORD MatchPatternTrail?
 
-MatchPattern ::= PlainQualifierExpression MatchPatternTrail?
+MatchPattern ::= PlainQualifierExpression MatchPatternTrail? MatchPatternDefault?
 private MatchPatternTrail ::= PAREN_OPEN <<commaOrEolList DestructuringPart>>? PAREN_CLOSE
+MatchPatternDefault ::= 'default' PAREN_OPEN {(AMPERSAND? MUT_KEYWORD?)? IDENTIFIER (EQUALS Expression)?} PAREN_CLOSE
```

Comment:
Keywords should be added to the `.flex` file and referred to in this file as a token. Be sure to add it to the syntax highlighter as well. 

Also (see previous comment about the curly braces):
```suggestion
MatchPatternDefault ::= 'default' PAREN_OPEN {AMPERSAND? MUT_KEYWORD? IDENTIFIER [EQUALS Expression]} PAREN_CLOSE
```

Reply:
`default` isn't a keyword though, it's a context-dependent name
in this case we can make it a keyword though, since it's not usable as a normal name anyway.